src/securityPolicy.c

```diff
@@ -69,22 +69,23 @@ security_policy_t policyForDerivePrivateKey(const bip44_path_t* path)
 	case PATH_ORDINARY_ACCOUNT:
 	case PATH_ORDINARY_SPENDING_KEY:
 	case PATH_ORDINARY_STAKING_KEY:
-
 	case PATH_MULTISIG_ACCOUNT:
 	case PATH_MULTISIG_SPENDING_KEY:
 	case PATH_MULTISIG_STAKING_KEY:
-
 	case PATH_MINT_KEY:
-
 	case PATH_POOL_COLD_KEY:
-
 		ALLOW();
+		break;
 
 	default:
 		DENY();
+		break;
 	}
+
+	DENY(); // should not be reached
```

Comment:
I know that editing this function was not necessary, I just wanted to make it visually aligned with `policyForGetExtendedPublicKey` and `policyForGetExtendedPublicKeyBulkExport` below.